libsolidity/codegen/CompilerUtils.cpp

```diff
@@ -174,15 +174,16 @@ void CompilerUtils::storeInMemoryDynamic(Type const& _type, bool _padToWordBound
 	else
 	{
 		unsigned numBytes = prepareMemoryStore(_type, _padToWordBoundaries);
-		if (numBytes > 0)
-		{
-			solUnimplementedAssert(
-				_type.sizeOnStack() == 1,
-				"Memory store of types with stack size != 1 not implemented."
-			);
-			m_context << Instruction::DUP2 << Instruction::MSTORE;
-			m_context << u256(numBytes) << Instruction::ADD;
-		}
+		solAssert(numBytes > 0,
```

Comment:
Good change!

Reply:
I think...

Reply:
I don't even remember why I wrote this...

Reply:
...just await the full result please.